pkg/cli/root.go

```diff
@@ -47,6 +47,7 @@ func NewRootCmd() *cobra.Command {
 					panic(errors.ErrBadConfig)
 				}
 			}
+			config.Extension.Cve = api.NewCveConfig(config.Extension.DBPath, config.Log)
```

Comment:
Why not make this a part of NewConfig() itself?

Reply:
I thought our plan is to make CveSearch as a different service, its good to have different method for CveConfig and use wherever we need as of now.